scripts/build-dependencies.sh

```diff
@@ -1,9 +1,11 @@
 #!/bin/bash
 
 # SUMMARY
-#   This script integrates secrets, regenerates Readium headers, wipes
-#   the Carthage folder, and finally checks out and rebuilds all Carthage
-#   dependencies.
+#   This script integrates secrets, regenerates Readium headers, and prepares
+#   the set for the Carthage build. If this script is called to build the app
+#   woth DRM support from a CI context, it simply ends here. Otherwise,
```

Comment:
Do you mean `without DRM` here?

Reply:
haha I do... great catch

Reply:
wait no I mean `with`! I'll fix it, the whole sentence is confusing.